test/distributed.cpp

```diff
@@ -17,7 +17,7 @@
 #include <fstream>
 #include <sstream>
 
-#include <mpi.h>
+#include <mlsl.hpp>
```

Comment:
This will require changes in CI. @lnguyen-nvn who could help us with this?

Reply:
I've checked failed CI build and there is a need to update the job to support MLSL. 
If the job executes `make install` then the following command should be executed before running the tests:
`source /path/to/install_dir/bin/mlslvars.sh`
Otherwise, if only `make` is executed by the job, then the following command should be added before running the tests:
`source /path/to/ngraph/build_dir/MLSL/install/intel64/bin/mlslvars.sh`
`mlslvars.sh` updates both PATH and LD_LIBRARY_PATH with mlsl/MPI libs and bins. Also I thinks it would be better to remove OpenMPI from the env.
@lnguyen-nvn could you please check and update the job?

Reply:
@rsketine : would you mind to take a look at this ? Thanks

Reply:
@rsketine could you please tell if you need additional information for this activity?